crates/viewer/re_renderer/src/video/chunk_decoder.rs

```diff
@@ -137,29 +137,33 @@ impl VideoChunkDecoder {
         let frame_idx = 0;
         let frame = &frames[frame_idx];
 
-        let frame_time_range = frame.info.presentation_time_range();
+        let frame_presentation_timestamp = frame.info.presentation_timestamp;
+
+        let gpu_texture = video_texture.texture.get_or_insert_with(|| {
+            alloc_video_frame_texture(
+                &render_ctx.device,
+                &render_ctx.gpu_resources.textures,
+                frame.content.width(),
+                frame.content.height(),
+            )
+        });
 
         let is_up_to_date = video_texture
             .frame_info
             .as_ref()
-            .is_some_and(|info| info.presentation_time_range() == frame_time_range);
+            // We assume that every frame has a unique presentation timestamp.
```

Comment:
I don't understand this. There is only one `frame.info.presentation_timestamp` and it is not a collection… so it's not much of an assumtion?

Reply:
should say `uniquely identified` :lightbulb: